calc/src/mechanics/gen8.ts

```diff
@@ -702,7 +702,7 @@ function calculateGen8(
     move.category = attackSource.stats.atk > attackSource.stats.spa ? 'Physical' : 'Special';
   }
   const attackStat =
-    move.category === 'Special' ? 'spa' : move.name === 'Body Press' ? 'atk' : 'def';
+    move.category === 'Special' ? 'spa' : move.name === 'Body Press' ? 'def' : 'atk';
```

Comment:
For readability reasons, we might as well do something like:

```suggestion
    move.name === 'Body Press' ? 'def' : (move.category === 'Special' ? 'spa' : 'atk');
```